packages/core/src/host.ts

```diff
@@ -26,7 +26,12 @@ export enum LogLevel {
     Error = 4,
 }
 
-export type APIType = "openai" | "azure" | "localai" | "azure_serverless"
+export type APIType =
+    | "openai"
+    | "azure"
+    | "localai"
+    | "azure_serverless"
+    | "anthropic"
```

Comment:
the apitype is really only used for variants of openai; since we have a specific provider here, i think we don't need this one.